docs/generator_template.md

```diff
@@ -0,0 +1,569 @@
+---
+title: "How to create a simple generator template"
+weight: 170
+---
+
+In this tutorial, you'll learn to create a simple generator template using a Python MQTT client. You'll use the AsyncAPI document and the template you develop to generate Python code. Additionally, you'll create template code with a reusable component to reuse the custom functionality you create and test your code using an MQTT client.
+
+Suppose you can only sleep when the AC in your bedroom is set to 22 °C, and you can't see when the temperature drops or rises above that. You can install a smart monitor in your bedroom that keeps track of the temperature and notifies you to adjust it to your optimum temperature if it fluctuates. You will create a template that alerts you when the temperature fluctuates from 22 °C.
```

Comment:
btw, love this idea, I felt like a motivation kick when I read it, to continue reading more. Great story based tutorial writing 💪🏼

Reply:
Thank you @derberg